app/templates/single_resource/index.html

```diff
@@ -20,8 +20,8 @@ <h2 class="ui dividing header">Resources</h2>
                             <a href="{{ url_for('single_resource.edit', resource_id=resource['id']) }}">{{ resource['name'] }}</a>
                         </td>
                         <td>{{ resource['address'] }}</td>
-                        <td>{{ resource['latitude'] | round }}</td>
-                        <td>{{ resource['longitude'] | round }}</td>
+                        <td>{{ resource['latitude'] }}</td>
```

Comment:
Yoni removed the round calls because some resources don't have lat/long values and this was causing a "Rounding none" error, I believe.